lib/services/ioc_container.dart

```diff
@@ -3,26 +3,26 @@ import 'package:zachranobed/routes/app_router.dart';
 import 'package:zachranobed/services/auth_service.dart';
 import 'package:zachranobed/services/box_movement_srvice.dart';
 import 'package:zachranobed/services/box_service.dart';
-import 'package:zachranobed/services/canteen_service.dart';
-import 'package:zachranobed/services/charity_service.dart';
 import 'package:zachranobed/services/delivery_service.dart';
-import 'package:zachranobed/services/fcm_token_service.dart';
+import 'package:zachranobed/services/entity_pairs_service.dart';
+import 'package:zachranobed/services/entity_service.dart';
 import 'package:zachranobed/services/offered_food_service.dart';
 import 'package:zachranobed/services/shipping_of_boxes_service.dart';
 
 class IoCContainer {
```

Comment:
Detail - I would move to `DI` directory and modify the name to` ioc_dependency_container` to keep the names consistent?

Reply:
Sure, renamed to `ServicesDependencyContainer` to be more specific and moved to "di" subdirectory 👌